src/StaticWebAssetsSdk/Tasks/Data/StaticWebAsset.cs

```diff
@@ -249,7 +249,7 @@ internal static (string fingerprint, string integrity) ComputeFingerprintAndInte
 
         internal static string ComputeIntegrity(string identity, string originalItemSpec)
         {
-            var file = File.Exists(identity) ?
+            using var file = File.Exists(identity) ?
```

Comment:
@javiercn When using the locally-built SDK, I wasn't able to reproduce the issue where the file was left in use between hot reload edits. But I've gone ahead and made this change anyway in case it was the culprit.

Reply:
I also confirmed that we do take this code path when making CSS changes:
https://github.com/dotnet/sdk/blob/22ea878bf0d9b285202fccfd8cbaa154dadac236/src/StaticWebAssetsSdk/Tasks/DefineStaticWebAssets.cs#L183
Which means that this extra `using` theoretically shouldn't make a difference.

But it could have been that for some reason this path was being taken earlier, when I observed that hot reload was breaking. And now that the other code path is taken, everything works.